### ISSUE TITLE
Add option for AM/PM

### ISSUE THREAD

**Issue by bdreager** [Regaerd/PyClock#17]

**Comment by livibetter** [comment]
I like the idea, thought I am 24h format person, but some people like 12h+AM/PM.

Here is a few thoughts I have
- I noticed you added a few issues for new features, are they part of
  v1.0.0, I hope not, since you already tagged. Either way, I think we
  can utilize Roadmap to help track what we really want for next
  version.
- Some of the issues are directly related the date/time format string.
  I think you need to outline the exactly all the date/time
  functionality/feature and code at once (not literally), so there
  wouldn't be foobar here to patch a date, dummy there to workaround
  something foobar does. It's more like a guideline and we can have the
  whole picture in mind while coding.
- Also about the AM/PM, I use en_us locale, and I am sure some locale
  won't use "AM" and "PM", so the font definitely can't render those.
  This will need to take into consideration to make sure the date/time
  string can be rendered properly, which I believe it's regardless the
  locale, users always get "AM"/"PM" style.

**Comment by bdreager** [comment]
> thought I am 24h format person, but some people like 12h+AM/PM.

Yeah, I don't have a use for it myself, but it will be nice for the sake of completeness.

> are they part of v1.0.0

Nah, I'm thinking most of the planned stuff will be at least v1.1.0.

> I think you need to outline the exactly all the date/time functionality/feature

I was going to keep the date format separate from the time format. Since it'll just be plaintext, it can be formatted however the user wants. Whereas the time format is very specific.

> This will need to take into consideration to make sure the date/time
> string can be rendered properly, which I believe it's regardless the
> locale

I'm thinking it'll just be an optional bool to turn the feature on and it'll just be AM/PM in plaintext. Nothing fancy that can break.

**Comment by livibetter** [comment]
> I was going to keep the date format separate from the time format. Since it'll just be plaintext, it can be formatted however the user wants. Whereas the time format is very specific.

I see, I was wondering if date would be another big text.

> I'm thinking it'll just be an optional bool to turn the feature on and it'll just be AM/PM in plaintext. Nothing fancy that can break.

Good, I was thinking too much.

**Comment by bdreager** [comment]
> I was wondering if date would be another big text.

Maybe if someone requests it. For now it'll just be the bare minimum.
